src-built-in/components/toolbar/stores/searchStore.js

```diff
@@ -59,17 +59,19 @@ var Actions = {
 			if (!menuWindow) return;
 			return menuWindow.isShowing(function (showing) {
 				if (showing) return;
-				let params = {
-					monitor: 'mine',
-					position: 'relative',
-					left: target.getBoundingClientRect().left,
-					forceOntoMonitor: true,
-					top: 'adjacent',
-					spawnIfNotFound: true
-				};
+
+				// TODO: Should we keep this to use later, or remove it?
+				// let params = {
+				// 	monitor: 'mine',
+				// 	position: 'relative',
+				// 	left: target.getBoundingClientRect().left,
+				// 	forceOntoMonitor: true,
+				// 	top: 'adjacent',
+				// 	spawnIfNotFound: true
+				// };
 
 				var bounds = document.getElementById("inputContainer").getBoundingClientRect();
-				menuWindow.showAt(window.screenX + bounds.left, target.offsetHeight + window.screenY, null, function (err, data) {
+				menuWindow.showAt(window.screenX + bounds.left, bounds.bottom + window.screenY, null, function (err, data) {
```

Comment:
@NinjaManatee I think we should be using LauncherClient.showWindow(top:"adjacent")

We should also be using WindowClient.isMouseOverDOMElement() instead of mouseInBounds.

Ugh, just kills me when we don't use our own convenience functions...

Reply:
@NinjaManatee if inputContainer isn't there for some reason you're going to get a nasty javascript exception.

Reply:
@thorsent re: showWindow, it was auto-focusing causing issues with the search menu's animations and stuff. Ryan used showAt to circumvent that problem.

Reply:
@thorsent @carterbs I think I've addressed your comments. I don't want to change to using showWindow because of the issue Brad mentioned. I've added a comment for future clarity.